next/components/forms/segments/AccountSections/ThankYouSection/ThankYouCard.tsx

```diff
@@ -44,13 +46,16 @@ const ThankYouCard = ({
         <h2 className="text-h2 text-center">{title}</h2>
         <AccountMarkdown variant="sm" content={content} />
       </div>
-      <div className="flex w-full flex-col items-center gap-4 px-0 sm:flex-row md:px-24">
```

Comment:
Have you please tested this change doesn't brake anything? It's used on multiple places in various configurations.

Reply:
I tested it, but just now realized we only want to change the feedback button for tax form. Pushed a new commit that does that